components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -474,7 +474,17 @@ public enum ErrorMessages {
         ERROR_CODE_ERROR_FIRING_EVENTS("65090", "Error while firing the events",
                 "Server encountered an error while firing the events"),
         ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_DEPTH("65091", "Unable to retrieve the organization depth.",
-                "Server encountered an error while retrieving depth of organization with ID: %s.");
+                "Server encountered an error while retrieving depth of organization with ID: %s."),
+        ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_APPLICATIONS("65092", "Error while retrieving applications",
```

Comment:
End the error messages and descriptions with a fullstop

Reply:
Fixed with 4b1995d272dccc95fc480070cea4e2f0d297b719